src/libp2p/service.rs

```diff
@@ -826,8 +826,7 @@ async fn emit_event(sender: &Sender<NetworkEvent>, event: NetworkEvent) {
 pub fn build_transport(local_key: Keypair) -> anyhow::Result<Boxed<(PeerId, StreamMuxerBox)>> {
     let build_tcp = || libp2p::tcp::tokio::Transport::new(libp2p::tcp::Config::new().nodelay(true));
     let build_dns_tcp = || libp2p::dns::TokioDnsConfig::system(build_tcp());
-    let transport =
-        libp2p::websocket::WsConfig::new(build_dns_tcp()?).or_transport(build_dns_tcp()?);
+    let transport = build_dns_tcp()?;
```

Comment:
So, we are not using WS anymore? Do you think this is fine for communicating with other peers?

Reply:
WS is only useful when TCP is disabled or not available (e.g. WASM nodes that run in browsers), which is not the case (I don't think lotus offers an option to disable TCP) and none of the bootstrap nodes use WS address

Reply:
Alright, in case it turns out we need it, we can re-enable it again.